servers/rendering/rendering_device.h

```diff
@@ -850,6 +850,8 @@ class RenderingDevice : public RenderingDeviceCommons {
 
 	enum StorageBufferUsage {
 		STORAGE_BUFFER_USAGE_DISPATCH_INDIRECT = 1,
+		STORAGE_BUFFER_USAGE_READONLY = 2,
+		STORAGE_BUFFER_USAGE_WRITEONLY = 4
```

Comment:
```suggestion
		STORAGE_BUFFER_USAGE_WRITEONLY = 4,
```
To make future extensions easier (like the first value)